Lombiq.Vsix.Orchard/Extensions/DTEExtensions.cs

```diff
@@ -2,6 +2,10 @@ namespace EnvDTE
 {
     public static class DteExtensions
     {
-        public static bool SolutionIsOpen(this DTE dte) => dte.Solution.IsOpen;
+        public static bool SolutionIsOpen(this DTE dte)
+        {
+            Microsoft.VisualStudio.Shell.ThreadHelper.ThrowIfNotOnUIThread();
```

Comment:
Ez OK-nak tűnik, de miért van rá szükség? Mikor lehetünk itt nem UI threaden, amikor jó megoldás exceptiont dobni? `DependencyInjector.Inject`, `GetExpectedClassName`, `GetCodeLines`, `UpdateCodeEditorAndSelectDependency` szintén.

Reply:
Azért volt rá szükség mert különben ez a warning jelenik meg:
![image](https://user-images.githubusercontent.com/18496537/155125847-25d4bfbe-bb21-4d44-ba4b-0a3e49629f20.png)

Reply:
Aha, érdekes. Akkor kérlek commenteld mindegyik ilyen helyre, hogy kb. "We should never get an exception here. This is just to make sure we access DTE on the main thread and get rid of the VSTHRD010 violation."